libs/template/src/lib/core/utils/make-zone-less.ts

```diff
@@ -0,0 +1,52 @@
+import { apiZonePatched, getGlobalThis } from '@rx-angular/template';
+
+/**
+ * getZoneUnPatchedApi
+ *
+ * @description
+ *
+ * This function returns the zone un-patched API for the a specific Browser API.
+ * If no element is passed the window is used instead
+ *
+ * @param name {string} - The name of the API to check.
+ * @param elem {any} - The elem to get un-patched API from.
+ * @return {Function} - The zone un-patched API in question.
+ *
+ */
+export function getZoneUnPatchedApi(name: string, elem?: any): Function {
+  elem = elem || getGlobalThis();
+  return apiZonePatched(name) ? elem[name] : elem['__zone_symbol__' + name];
+}
+
+/**
+ * reapplyEventListenerZoneUnPatched
+ *
+ * @description
+ *
+ * This function takes an elem and event and re-applies the listeners from the passed event to the
+ * passed element with the zone un-patched version of it.
+ *
+ * @param elem {HTMLBaseElement} - The elem to re-apply the listeners to.
+ * @param event {string} - The name of the event from which to re-apply the listeners.
+ *
+ */
+export function reapplyEventListenerZoneUnPatched(
+  elem: HTMLBaseElement,
```

Comment:
i am pretty sure u want `HTMLElement`, not HTMLBaseElement

Reply:
Yes!